src/components/RemoveLiquidityModal/RemoveLiquidityModal.tsx

```diff
@@ -222,12 +224,12 @@ const RemoveLiquidityModal: React.FC<RemoveLiquidityModalProps> = ({
     };
 
     const liquidityAmount = parsedAmounts[Field.LIQUIDITY];
-    if (!liquidityAmount) throw new Error('missing liquidity amount');
+    if (!liquidityAmount) throw new Error(t('noLiquidity'));
 
     const currencyBIsETH = currency1 === ETHER;
     const oneCurrencyIsETH = currency0 === ETHER || currencyBIsETH;
 
-    if (!tokenA || !tokenB) throw new Error('could not wrap');
+    if (!tokenA || !tokenB) throw new Error('cannotWrap');
```

Comment:
I think you meant to use the translation in the error message

Reply:
Fixed this and pushed another commit